chatmaild/src/chatmaild/metrics.py

```diff
@@ -11,6 +11,8 @@ def main(vmail_dir=None):
     ci_accounts = 0
 
     for path in Path(vmail_dir).iterdir():
+        if not path.is_dir() or not (path / "cur").is_dir():
```

Comment:
it's sufficient to directly test the target dir i think 

```suggestion
        if not (path / "cur").is_dir():
```